docs/quickstart.md

```diff
@@ -23,7 +23,49 @@ Or, to import an existing Solidity project into Web3j, run:
 ```bash
 web3j import
 ```
-
+​
+The first step after creating a new project is to fund your wallet.
+The wallet's address will be printed on the console after the project is created.
+​
+To fund your wallet you can go to `https://rinkeby.faucet.epirus.io/` and paste in your wallet address.
+​
+After your wallet is funded there are 3 ways to execute your code.
+​
+#### Using Gradle run to execute your code:
+​
+One way you can execute your code is by running `./gradlew run`.
+​
+When executing your application with ./gradlew run you need to set up your Ethereum node URL as an environmental variable as such:
+​
+- `export NODE_URL= https://example.com` 
```

Comment:
Is example.com a good one or should we use something like rinkeby.infura.io/v3/token?